example/flutter_example/pubspec.yaml

```diff
@@ -0,0 +1,47 @@
+name: flutter_example
+description: toml flutter example
+
+# The following line prevents the package from being accidentally published to
+# pub.dev using `pub publish`. This is preferred for private packages.
+publish_to: 'none' # Remove this line if you wish to publish to pub.dev
+
+# The following defines the version and build number for your application.
+# A version number is three numbers separated by dots, like 1.2.43
+# followed by an optional build number separated by a +.
+# Both the version and the builder number may be overridden in flutter
+# build by specifying --build-name and --build-number, respectively.
+# In Android, build-name is used as versionName while build-number used as versionCode.
+# Read more about Android versioning at https://developer.android.com/studio/publish/versioning
+# In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
+# Read more about iOS versioning at
+# https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
+version: 1.0.0+1
```

Comment:
Is the `version` field required by Flutter or can it be removed since it is a [local-only package](https://dart.dev/tools/pub/pubspec#version)?

Reply:
I dont think its needed, i tested it to run ok on both iOS and Android after i commented it out. I removed the other comments above.

Reply:
I looked at some examples for official flutter plugins and they dont use it, so i removed it in an additional commit and cleaned up the pubspec.yaml file somewhat.